### DISCUSSION TITLE
How to reduce spaces between blocks ?

### DISCUSSION THREAD

**Discussion by Bebio95** [datapane/datapane#92] [Help]
Hello, 

I am wondering if there is a way to reduce the spaces between blocks of text and/or plots ?
I join an image to illustrate the issue.

Thanks for any answers you could have
<img width="1432" alt="Capture d’écran 2021-03-12 à 20 05 39" src="https://user-images.githubusercontent.com/80545003/110988870-62610280-8371-11eb-951b-c650a2a536ac.png">

**Comment by andrewhong5297** [comment]
same question here, but in column spacing! would like to reduce space between columns and fill the page

**Reply by johnmicahreid** [reply]
Hi @Bebio95, if you have several continuous blocks of text, you could combine them into a single Markdown block:  https://docs.datapane.com/reports/blocks/text-code-and-html#text
@andrewhong5297 you can change the default column widths by selecting the report type: https://docs.datapane.com/reports/blocks#report-types. 
We'll be looking into offering more ways to customize the margins and spacing, thanks for this feedback!

**Comment by isergii** [comment]
Hi @johnmicahreid , @mands ,

Any updates regarding margins and spacing flexibility in layout (current margins are so big)? I couldn't find any workaround here.

Thanks.

**Reply by johnmicahreid** [reply]
Hi @isergii, we're working on changing the defaults for padding and margins so the reports look better, expect a new version in about a month or so. Customizing margins/padding on a per-block level is also on our long-term roadmap!

**Reply by isergii** [reply]
Great! thanks for clarification.

**Reply by johnmicahreid** [reply]
@isergii @andrewhong5297 @Bebio95 we've pushed a new version where we've updated some of the defaults and reduced the margins - could you let us know how it looks now?

**Reply by isergii** [reply]
Thanks @johnmicahreid. I will take a look soon.
